flight_phase_files/HALO_RF14_20200215_info.yaml

```diff
@@ -0,0 +1,73 @@
+name: RF14
+mission: EUREC4A
+flight_id: HALO-0215
+contacts:
+- name: Marc Prange
+  email: marc.prange@uni-hamburg.de
+  tags:
+  - dataset creator
+- name: Sandrine Bony
+  email: bony@lmd.jussieu.fr
+  tags:
+  - flight PI
+date: 2020-02-15
+flight_report: https://observations.ipsl.fr/aeris/eurec4a-data/REPORTS/HALO/2020/20200215/HALO-0215.pdf
+takeoff: 2020-02-15 15:07:30
+landing: 2020-02-16 00:12:44
+events: []
+remarks:
+- First two circles near Barbados. Excursion to NTAS. Third circle at NTAS location.Last
+  three circles again near Barbados.
+- Fewer dropsondes launched than usual because of the availibility at this stage of
+  the campaign.
+- '**Only one dropsonde during circle 5!**'
+- Thick and persistent altostratus layer (top at ~9km) covering the area from Barbados
+  to NTAS.
+- Beneath the altostratus layer shallow clouds and stratiform layers were present.
+- Flight went above and below the altostratus layer.
+- '**See flight-report for more information.**'
+segments:
+- kind: circle
+  name: c1
+  start: 2020-02-15 15:38:41
+  end: 2020-02-15 16:41:39
+- kind: circle_break
+  name: cb1to2
+  start: 2020-02-15 16:41:39
+  end: 2020-02-15 17:00:13
+- kind: circle
+  name: c2
+  start: 2020-02-15 17:00:13
+  end: 2020-02-15 17:41:55
+- kind: straight_leg
+  name: leg1_outbound_NTAS
+  start: 2020-02-15 17:52:34
+  end: 2020-02-15 18:15:01
+- kind: circle
+  name: c3
+  start: 2020-02-15 18:17:29
+  end: 2020-02-15 19:16:33
+- kind: straight_leg
+  name: leg2_inbound_NTAS
+  start: 2020-02-15 19:28:05
+  end: 2020-02-15 19:59:51
+- kind: circle
+  name: c4
+  start: 2020-02-15 20:10:03
+  end: 2020-02-15 21:20:20
+- kind: circle
+  name: c5_no_dropsondes
```

Comment:
Name this c5, there are 11 sondes, when counting the one before the unplanned circle deviation towards c4.